support/docker/production/Dockerfile.arm

```diff
@@ -0,0 +1,65 @@
+FROM arm64v8/debian:stretch-slim
+
+RUN set -ex; \
+    if ! command -v gpg > /dev/null; then \
+      apt-get update; \
+      apt-get install -y --no-install-recommends \
+        gnupg \
+        dirmngr \
+      ; \
+      rm -rf /var/lib/apt/lists/*; \
+fi
+
+# install nodejs and npm
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update \
+&& apt-get install -y apt-utils nano htop curl git build-essential gcc g++ make \
+&& curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+&& apt-get install -yq nodejs \
+&& rm -rf /var/lib/apt/lists/ \
+&& npm install npm@latest -g && npm install yarn@latest -g
+
+# Install dependencies
+RUN apt-get update \
+    && apt-get -y install ffmpeg \
+    && rm /var/lib/apt/lists/* -fR
+
+# Add peertube user
+RUN groupadd -r peertube \
+    && useradd -r -g peertube -m peertube
+
+# grab gosu for easy step-down from root
+RUN set -eux; \
+	apt-get update; \
+	apt-get install -y gosu; \
+	rm -rf /var/lib/apt/lists/*; \
+	gosu nobody true
+
+# Install PeerTube
+WORKDIR /app
+COPY . ./
+RUN chown -R peertube:peertube /app
+
+USER peertube
+# from: https://github.com/yarnpkg/yarn/issues/5259#issuecomment-421067226
+RUN for attempt in $(seq 1 100); do yarn install --pure-lockfile --verbose && break; done
+RUN npm run build && rm -r ./node_modules ./client/node_modules
```

Comment:
yarn run build ?

Reply:
It seems I was not on the correct branch but there is this command used in the original dockerfile.

Replaced by `&& npm run build -- $NPM_RUN_BUILD_OPTS \` later.

Reply:
that doesn't realy change somethings, -- means pass this to the script and yarn also handle that.

Reply:
Done in all dockerfiles (I keep the argument name for other automation tools)